handlers.js

```diff
@@ -0,0 +1,102 @@
+const isEqual = require('lodash.isequal');
+
+module.exports = self => {
+  return {
+    'apostrophe:modulesRegistered': {
+      async addMissingPieces() {
```

Comment:
This method needs to use an apostrophe lock so that separate instances do not collide when they execute it